lib/cell/assets_paths.rb

```diff
@@ -0,0 +1,33 @@
+module Cell::AssetsPaths
```

Comment:
Do you think we can "merge" that with the view prefix logic, or is your code diverging from the algorithm there?